Doc/whatsnew/3.11.rst

```diff
@@ -176,14 +176,33 @@ The :option:`-X` ``no_debug_ranges`` option and the environment variable
 See :pep:`657` for more details. (Contributed by Pablo Galindo, Batuhan Taskaya
 and Ammar Askar in :issue:`43950`.)
 
+Exception Groups and ``except*`` (PEP 654)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+:pep:`654` introduces language features that enable a program
+to raise and handle multiple unrelated exceptions simultaneously.
+The builtin types :exc:`ExceptionGroup` and :exc:`BaseExceptionGroup`
+make it possible to group exceptions and raise them together.
+The new ``except*`` syntax generalizes ``except`` to match subgroups
+of exception groups.
+
+See :pep:`654` for more details.
+
+(Contributed by Irit Katriel in :issue:`45292`. PEP written by
+Irit Katriel, Yury Selivanov and Guido van Rossum.)
+
 Exceptions can be enriched with notes (PEP 678)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 The :meth:`add_note` method was added to :exc:`BaseException`. It can be
 used to enrich exceptions with context information which is not available
 at the time when the exception is raised. The notes added appear in the
-default traceback. See :pep:`678` for more details. (Contributed by
-Irit Katriel in :issue:`45607`.)
+default traceback.
+
+See :pep:`678` for more details.
+
+(Contributed by Irit Katriel in :issue:`45607`. PEP written by
+Zac Hatfield-Dodds.)
```

Comment:
Could we make these changes in #95915 instead, since that PR is closer to the scope of the change (copyediting existing entries vs. adding a specific new one), and it will cause a significant merge conflict for whichever PR gets merged second otherwise?

Reply:
ok, I'll remove it and you can put it there.